src/theme/material/templates/plugin-eb-autocomplete.tpl.html

```diff
@@ -9,18 +9,19 @@
 					matTooltipShowDelay="500"
 					matTooltipClass="q-grid-query-builder-error-tooltip">
 		<input matInput
-		       #input
+		       #input="matAutocompleteTrigger"
 		       [(ngModel)]="$expr.value"
+				 (click)="input.openPanel()"
 		       (input)="$expr.change && $expr.change();$expr.refresh && $expr.refresh()"
 		       (optionSelected)="$expr.change && $expr.change()"
 		       [placeholder]="$expr.placeholderText"
 		       [matAutocomplete]="auto">
-	</mat-form-field>
 
-	<mat-autocomplete #auto="matAutocomplete">
-		<mat-option *ngFor="let option of $expr.options | async"
-		            [value]="option">
-			{{option}}
-		</mat-option>
-	</mat-autocomplete>
-</ng-template>
\ No newline at end of file
+		<mat-autocomplete #auto="matAutocomplete" (q-grid-backdrop)="input.closePanel()" [q-grid-backdrop-element]="auto" [q-grid-backdrop-is-active]="auto.isOpen">
```

Comment:
rename #auto to #autoComplete

Reply:
also use `[q-grid-backdrop-host]="autoComplete.panel"` instead of `[q-grid-backdrop-element]="auto"`